backend/schema/message.py

```diff
@@ -81,3 +81,13 @@ class Message(Thread):
     """
 
     threads: List[Thread] = []
+
+
+class MessageUpdate(BaseModel):
```

Comment:
I thought about using the MessageRequest class initially but I wasn't too keen on editing someone else's code. However I can simply remove the edited at field, that should work too

Reply:
@mykie88 I took out the edited_at field. 
@o4codes unfortunately I did not adjust the endpoint to use the messagerequest class because it would require me editing that class. Re the  method fastapi provides, maybe you could set up a meet and walk me through.